### PR TITLE
Document chickenshock-backend architecture

### PR DESCRIPTION
**Pull request by LeviUniAcc**

### CODE REVIEW

File: dev-manuals/architecture/chickenshock-backend/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+- `data`: Contains the data objects and DTOs
+- `controllers`: Containers the server routes
+- `services`: Contains the business logic
+- `clients`: Contains the b2b (backend-2-backend) communication methods
+- `repositories`: Contains the database repositories
+
+
+## Starting the service
+
+See the README or the user manual.
```
Comment (delvh) [inline dev-manuals/architecture/chickenshock-backend/README.md:24]:
```suggestion
See the [README](https://github.com/Gamify-IT/chickenshock-backend#readme).
```

File: dev-manuals/architecture/chickenshock-backend/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## What to look at
+
+- every data-class has an additional DTO class for savety purposes
+
+## What to ignore
+
+- nothing
```
Comment (delvh) [inline dev-manuals/architecture/chickenshock-backend/README.md:32]:
```suggestion

_Everything_

## What to ignore

_NA_
```

File: dev-manuals/architecture/chickenshock-backend/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+- `repositories`: Contains the database repositories
+
+
+## Starting the service
+
+See the README or the user manual.
+
+## What to look at
+
+- every data-class has an additional DTO class for savety purposes
```
Comment (maexled) [inline dev-manuals/architecture/chickenshock-backend/README.md:28]:
```suggestion
- every entity class has an additional DTO class for safety purposes
```

Comment (maexled) [reply dev-manuals/architecture/chickenshock-backend/README.md:28]: Why explicitly look at the DTOs?

Comment (delvh) [reply dev-manuals/architecture/chickenshock-backend/README.md:28]: See also https://github.com/Gamify-IT/docs/pull/128#discussion_r955134299

Comment (LeviUniAcc) [reply dev-manuals/architecture/chickenshock-backend/README.md]: idk i thought it was atleast a little bit special... we can remove it if you want

Comment (LeviUniAcc) [reply dev-manuals/architecture/chickenshock-backend/README.md]: oh ok it already is removed

**Review by delvh — APPROVED** [review]

**Pull request closed without merging**